cumulus/README.md

```diff
@@ -145,6 +145,10 @@ git clone https://github.com/paritytech/polkadot-sdk
 # Compile Polkadot
 cargo build --release --bin polkadot
 
+# Compile Polkadot's `execute` and `prepare` binaries
+cargo build --release --bin polkadot-prepare
+cargo build --release --bin polkadot-execute
```

Comment:
Do we need the workers here? I thought they are only needed for validators. BTW, this looks a bit nicer (the parentheses start a subshell so we don't have to `cd ..` after):

```suggestion
(cd polkadot; cargo build --release)
```

Reply:
Yes, because you need to onboard your parachain somewhere.